t/t5403-post-checkout-hook.sh

```diff
@@ -56,11 +58,12 @@ test_expect_success 'post-checkout args are correct with git checkout -b ' '
 '
 
 test_expect_success 'post-checkout receives the right args with HEAD changed ' '
-	GIT_DIR=clone2/.git git checkout new2 &&
-	old=$(awk "{print \$1}" clone2/.git/post-checkout.args) &&
-	new=$(awk "{print \$2}" clone2/.git/post-checkout.args) &&
-	flag=$(awk "{print \$3}" clone2/.git/post-checkout.args) &&
-	test $old != $new && test $flag = 1
+	( cd clone2 &&
+	GIT_DIR=.git git checkout new2 &&
```

Comment:
Please do not set `GIT_DIR`. I think it does something different than what you expect.

Reply:
Done